### PR TITLE
feat: add shortcut for navigate code action menu

### PR DESCRIPTION
**Pull request by nix6839**
Support shortcut for navigate custom code action menu on VSCode >= 1.71.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1213,6 +1213,16 @@
                 "command": "vscode-neovim.complete-selection-cmdline",
                 "key": "tab",
                 "when": "neovim.mode == cmdline_normal || neovim.mode == cmdline_insert || neovim.mode == cmdline_replace"
+            },
+            {
+                "key": "ctrl+n",
```
Comment (justinmk) [inline package.json:1218]: is this analogous to `]q` from https://github.com/tpope/vim-unimpaired ? vim-unimpaired is a very well-established convention so I suggest we use its mappings, i.e. `]q` and `[q`.

**Review by justinmk — APPROVED** [review]
Great to eliminate these kinds of things from the readme in a simple way.

File: package.json (hunk 1 above)
Comment (theol0403) [reply package.json:1218]: No, this is for navigating quick pick (code actions mostly). However, connecting unimpaired bindings to vscode like `]q` to jump between LSP errors would be a great addition!

**Comment by theol0403** [comment]
Thanks for the PR!

**Pull request merged**